index.html

```diff
@@ -0,0 +1,732 @@
+    "userId":"testuser",
+    "issueKey":"TEST-1"
+  },
+  {
+    "id":11110,
+    "duration":17100,
+    "startDate":"2013-01-09T08:30:00Z",
+    "userId":"testUser2",
+    "issueKey":"TEST-2"
+  }
+]]</code></pre>
+        <p>In this example, testUser and testUser2 are members of the test group. The issues
+          with the codes TEST-1 and TEST-2 belong to the TEST project.</p>
+        <h3 id="example3">Example 3</h3>
+        <p>If the authenticated user does not have permission to access the TEST project, the
+          response is an empty list: [[]]</p>
+        <h3 id="example4">Example 4</h3>
+        <pre><code>JIRA_URL/rest/jira-worklog-query/1/find/updatedWorklogs?startDate=2012-12-12&group=test group&project=TEST</code></pre>
+        <p>During the interval between 2012.12.12 and 2013.01.12, all the worklogs modified by
+          the testuser are returned from every project to which the user has access to.</p>
+        <p>A possible response:</p>
+        <pre><code>[[
+  {
+    "id":11000,
+    "duration":14400,
+    "startDate":"2013-01-07T08:00:00Z",
+    "userId":"testuser",
+    "issueKey":"TEST-1"
+  }
+]]</code>
+        </pre>
+        <p>The response is similar to the one in the first example, but the last worklog is
+          missing from the response. One possible explanation for that may be that the testuser
+          modified the worklog description to 2013.01.13. Therefore, the updateWorklogs query does
+          not return it any longer, while a worklog query would return the results of the first
+          example henceforward.</p>
+        <h3 id="example5">Example 5</h3>
+        <pre><code>JIRA_URL/rest/jira-worklog-query/1/find/worklogsByIssues?startDate=2015-01-10&endDate=2015-01-11&user=testuser</code></pre>
+        <p>A possible response:</p>
+        <pre><code>{
+  "startAt": 0,
+  "maxResults": 25,
+  "total": 2,
+  "issues": [
+    {
+      "id": "13405",
+      "self": "http://localhost:8080/rest/api/2/issue/13405",
+      "key": "TEST-1",
+      "timespent": 1440
+    },
+    {
+      "id": "13406",
+      "self": "http://localhost:8080/rest/api/2/issue/13406",
+      "key": "TEST-2",
+      "timespent": 35760
+    }
+  ]
+}</code></pre>
+        <p>Example response with "fields=summary,progress" parameter:</p>
+        <pre><code>{
+  "startAt": 0,
+  "maxResults": 25,
+  "total": 1,
+  "issues": [
+    {
+      "id": "13411",
+      "self": "http://localhost:8080/rest/api/2/issue/13411",
+      "key": "NEW-1",
+      "fields": {
+        "progress": {
+          "progress": 84000,
+          "total": 84000,
+          "percent": 100
+        },
+        "summary": "new issue1"
+      },
+      "timespent": 67080
+    }
+  ]
+}</code></pre>
+      </div>
+      <hr class="featurette-divider">
+
+      <div class="bs-docs-featurette cookbook" id="release_notes">
+        <h2>Releases</h2>
+        <a href="https://github.com/everit-org/jira-worklog-query-plugin/releases">RELEASES</a>
```

Comment:
Add a sentence, for example: The version history and the release notes can be found on the Releases page of the Github project.
The "Releases" word is a link.